### PR TITLE
Sort priorities on FixedPriority Strategy.

### PR DESCRIPTION
**Pull request by rafbgarcia**
FixedPriority returns the most priority agent.

### CODE REVIEW

**Comment by bklang** [comment]
Nice catch, thanks! :+1:

**Comment by benlangfeld** [comment]
Changelog?

**Comment by rafbgarcia** [comment]
Should I create a new PR for the Changelog?

**Comment by benlangfeld** [comment]
Yes please :) In future, that should block merging.

File: lib/electric_slide/agent_strategy/fixed_priority.rb [hunk 1]
```diff
@@ -30,7 +30,8 @@ def available_agent_summary
       end
 
       def checkout_agent
-        _, agents = @priorities.detect do |priority, agents|
+        priorities = @priorities.sort.to_h
```
Comment (benlangfeld) [inline lib/electric_slide/agent_strategy/fixed_priority.rb]: I don't think you need to cast this to a Hash at all, it should work fine without.

File: lib/electric_slide/agent_strategy/fixed_priority.rb [hunk 2]
```diff
@@ -30,7 +30,8 @@ def available_agent_summary
...
+        _, agents = priorities.detect do |priority, agents|
```
Comment (benlangfeld) [inline lib/electric_slide/agent_strategy/fixed_priority.rb]: I'd prefer to just inline the call to `#sort` here for clarity.

**Comment by benlangfeld** [comment]
My vote would have been -1 here pending the above comments.

**Pull request merged**